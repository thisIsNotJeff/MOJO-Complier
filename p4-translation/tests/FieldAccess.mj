var r: ( x: int; r: ( y: int ); z: int);
{
  r.x := 1;
  r.r.y := 2;
  r.z := 3;
}