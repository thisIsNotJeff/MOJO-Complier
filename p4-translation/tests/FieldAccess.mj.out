	.data
	.balign 8
FieldAccess.r:
	.quad 0
	.quad 0
	.quad 0
	.text
FieldAccess:
MOVE(
 MEM(
  NAME FieldAccess.r, 
  CONST 0, 8),
 CONST 1),
MOVE(
 MEM(
  NAME FieldAccess.r, 
  CONST 8, 8),
 CONST 2),
MOVE(
 MEM(
  NAME FieldAccess.r, 
  CONST 16, 8),
 CONST 3)


